content/docs/development/oidc-certification.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "OpenID Connect Provider Certification"
+description: ""
+date: 2020-09-30
+draft: false
+toc: true
+weight: 1100
+---
+
+The OpenID Foundation provides a set of [conformance test profiles](https://openid.net/wordpress-content/uploads/2018/06/OpenID-Connect-Conformance-Profiles.pdf) that test both Relying Party
+and OpenID Provider (OP) OpenID Connect implementations.
+Upon submission of [results](https://openid.net/certification/submission/) and an affirmative response,
+the affirmed OP will be listed as a [certified OP](https://openid.net/developers/certified/) on the OpenID Connect website
+and allowed to use the [certification mark](https://openid.net/certification/mark/) according to the certification [terms and conditions](https://openid.net/wordpress-content/uploads/2015/03/OpenID-Certification-Terms-and-Conditions.pdf), section 3(d).
+
+Further details about the certification are available on the [OpenID Connect website](https://openid.net/certification/instructions/).
+
+## Basic OpenID Provider profile
+
+Dex is an OP that strives to implement the [mandatory set](https://openid.net/specs/openid-connect-core-1_0.html#ServerMTI) of OpenID Connect features,
+and can be tested against the Basic OpenID Provider profile ([profile outline](https://openid.net/wordpress-content/uploads/2018/06/OpenID-Connect-Conformance-Profiles.pdf), section 2.1.1).
+These tests ensure that all features required by a [basic client](https://openid.net/specs/openid-connect-basic-1_0.html) work as expected.
+
+Unfortunately, Dex currently does not fully comply with the Basic profile at the moment.
+
+The progress for getting Dex certified can be tracked here: https://github.com/orgs/dexidp/projects/3/views/1
+
+### Configuring Dex
+
+The Basic OP test suite doesn't require extensive configuration from Dex.
+The suite needs the following:
+
+- A public issuer URL
+- At least two separate clients (with redirect URIs pointing to `https://www.certification.openid.net/test/a/YOUR_ALIAS/callback`).
+
+`YOUR_ALIAS` is an arbitrary string that MUST be unique to avoid interference with other test runs.
+
+The easiest way to run a public Dex instance is running one locally and exposing it using a [tunnel](https://github.com/anderspitman/awesome-tunneling).
+[Ngrok](https://ngrok.com/) is probably the most popular one. Personally, I prefer using [tunnelto.dev](https://tunnelto.dev/) these days.
```

Comment:
From my point of view, it would be better to stick to the `tunnelto.dev` and rewrite this block using the imperative (to make it sound more like instruction).

Reply:
Works for me. Will update the docs later.